app/src/test/java/invoice_automation/module/QuickBooksUtilsTest.java

```diff
@@ -0,0 +1,67 @@
+package invoice_automation.module;
+
+import com.intuit.ipp.data.Customer;
+import com.intuit.ipp.data.PhysicalAddress;
+import com.intuit.ipp.util.Config;
+import invoice_automation.model.Address;
+import invoice_automation.model.School;
+import invoice_automation.utils.QuickBooksUtils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({QuickBooksUtils.class, Config.class})
+public class QuickBooksUtilsTest {
+
+    @Test
+    public void getCustomerFromSchool_happyPath() {
+        // Setup
+        List<String> phoneNumbers = new ArrayList<>();
+        Address address = new Address("", "", "", "", "", "");
+        phoneNumbers.add("1234567890");
+        phoneNumbers.add("0987654321");
+        School school = School.builder()
+                .schoolName("Berkeley")
+                .quickBooksId("id")
+                .address(address)
+                .email("oski@berkeley.edu")
+                .phoneNumbers(phoneNumbers)
+                .build();
+
+        // Run
+        Customer customer = QuickBooksUtils.getCustomerFromSchool(school);
+
+        // Verify
+        assertEquals(school.getSchoolName(), customer.getDisplayName());
+        assertEquals(school.getSchoolName(), customer.getCompanyName());
+        assertEquals(school.getEmail(), customer.getPrimaryEmailAddr().getAddress());
+        assertEquals(school.getPhoneNumbers().get(0), customer.getPrimaryPhone().getFreeFormNumber());
+        assertEquals(school.getPhoneNumbers().get(1), customer.getAlternatePhone().getFreeFormNumber());
+        assertEquals(QuickBooksUtils.getPhysicalAddressFromAddress(address), customer.getBillAddr());
```

Comment:
nit: this doesn't test that the customer's address matches the school we gave, rather it tests that the function calls getPhysicalAddressFromAddress and sets the returned value